docs/index.md

```diff
@@ -2,4 +2,74 @@
+the Substrate Runtime Module Library ([SRML](conceptual/runtime/srml.md)) - regarding the underlying
+data-structures that power the STF, thereby making rapid blockchain development a reality.
+
+## Usage
+
+![Technical Freedom vs Development Ease](assets/technical-freedom.png)
+
+Substrate is designed to be used in one of three ways:
+
+1. **With the Substrate Node**: You can run the pre-designed Substrate Node and configure it with a
+   genesis block that includes the default node runtime. In this case, you just need to configure a
+   JSON file and launch your own blockchain. This affords you the least amount of customization,
+   only allowing you to change the genesis parameters of the included runtime modules such as:
+   balances, staking, block-period, fees, governance, etc... For a tutorial on doing this, see
+   [Start a Private Network with Substrate](tutorials/start-a-private-network-with-substrate.md).
+
+2. **With the SRML**: You can easily create your own custom blockchain using the SRML. This affords
+   you a very large amount of freedom over your own blockchain's logic, letting you change
+   datatypes, select from the library of modules, and add your own custom modules. Much can be
+   changed without touching the block-authoring logic since it is directed through on-chain logic.
+   If this is the case, then the existing Substrate binary can be used for block authoring and
+   syncing. If the block authoring logic needs to be modified, then a new block-authoring binary
+   must be built as a separate project and used by validators. This is how the Polkadot relay chain
+   is built and should suffice for almost all needs in the near future. For a tutorial on this, see
+   [creating your first Substrate chain](tutorials/creating-your-first-substrate-chain/index.md).
+
+3. **With the Substrate Core**: The entire [SRML](conceptual/runtime/srml.md) can be ignored, and
+   the entire runtime can be designed and implemented from scratch. This could be done in _any
+   language_ that can target WebAssembly. If the runtime can be made to be compatible with the
+   abstract block authoring logic of the Substrate node, then you can simply construct a new genesis
+   block from your Wasm blob and launch your chain with the existing Rust-based Substrate client. If
+   not, then you will need to alter the client's block authoring logic, and potentially even
+   alter the header and block serialization formats. In terms of development effort, this is by
+   far the most difficult way to use Substrate, but also gives you the most freedom to innovate.
+
+## Next Steps
+
+### Learn More
+
+- View our developer documentation for [using the SRML for module
+  development](development/module/index.md).
+
+### Examples
+
+- Follow our [tutorial to create your first custom blockchain with
+  Substrate](tutorials/creating-your-first-substrate-chain/index.md).
+
+- Follow our [tutorial to start a private network with
```

Comment:
How about we put this one as the first example? Since it doesn't require any programming and user can feel substrate.